maisim/maisim.Game/Component/Gameplay/Notes/Break.cs

```diff
@@ -0,0 +1,34 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osuTK;
+
+namespace maisim.Game.Component.Gameplay.Notes
+{
+    /// <summary>
+    /// Class represent the BREAK note.
+    /// </summary>
+    public class Break : MaisimNotes
+    {
+        [BackgroundDependencyLoader]
+        private void load(TextureStore textureStore)
+        {
+            InternalChild = new Container
```

Comment:
This container is being constructed for every single note type with the same settings, you may to move these settings instead to  `MaisimNotes` base class.